types/apollo-codegen/index.d.ts

```diff
@@ -1,13 +1,15 @@
-// Type definitions for apollo-codegen 0.16
+// Type definitions for apollo-codegen 0.20
 // Project: https://github.com/apollographql/apollo-codegen
-// Definitions by: Bradley Ayers <https://github.com/bradleyayers>
+// Definitions by: Bradley Ayers <https://github.com/bradleyayers>, Maria Carrasco <https://github.com/kostspielig>
```

Comment:
Not sure if comma-separated works here, in the past I've only seen new-line-separated.

Reply:
comma-separated works too

Reply:
Thanks!